src/js/sdk.tsx

```diff
@@ -276,3 +276,20 @@ export function withScope(callback: (scope: Scope) => void): ReturnType<Hub['wit
   };
   getCurrentHub().configureScope(safeCallback);
 }
+
+/**
+ * Captures a message event and sends it to Sentry.
+ *
+ * @param message The message to send to Sentry.
+ * @param Severity Define the level of the message.
+ * @returns The generated eventId.
+ */
+export function captureMessage(message: string, level?: SeverityLevel): string | undefined {
+  return getCurrentHub().getClient<ReactNativeClient>()?.captureMessage(
```

Comment:
When I make this change the `frames` start to disappear and I don't know why.

```suggestion
  return getCurrentHub().captureMessage(
```

Reply:
is it because the synthetic exception gets re-written?

Reply:
The `synthetic exception` gets all the way to the [`eventFromString`](https://github.com/getsentry/sentry-javascript/blob/e1d43ebb34086605aefb9243fce03fc97425b4be/packages/browser/src/eventbuilder.ts#L276), but there the `frames` are missing in the `stacktrace` even tho that are assigned to it.

Reply:
IDK if its the case, but in some languages/platforms, when you do `Error(message)`, the stack trace isn't available.
You'd need to `throw Error(message)` and only then access the stack trace from the thrown exception.

Reply:
The stack is present in any case it just disappears later in the [`eventFromString`](https://github.com/getsentry/sentry-javascript/blob/e1d43ebb34086605aefb9243fce03fc97425b4be/packages/browser/src/eventbuilder.ts#L276)

Reply:
@krystofwoldrich can you check if this has something to do with `DebugSymbolicator`?
Does it happen in release and debug mode? or only one of them?

Reply:
@marandaneto Wow, thank you, DebugSymbolicator is causing it. I haven't looked at why, yet.